src/Playwright.Tests/TestConstants.cs

```diff
@@ -33,17 +33,36 @@ namespace Microsoft.Playwright.Tests
 {
     internal static class TestConstants
     {
-        public static string BrowserName => NUnit.PlaywrightTest.BrowserName;
+        public static string BrowserName { get; set; } = null!;
 
         public const int DefaultTestTimeout = 30_000;
         public const int DefaultTimeout = 10_000;
         public const int DefaultTaskTimeout = 5_000;
 
-        public const string AboutBlank = "about:blank";
+        internal static bool IsWebKit
```

Comment:
Nit: you can use the shorthand syntax if you want:
```suggestion
        internal static bool IsWebKit => BrowserName == BrowserType.Webkit;
```

Reply:
sweet.